src/index.ts

```diff
@@ -445,6 +459,11 @@ function activateCellVim(app: JupyterLab, tracker: INotebookTracker): Promise<vo
             keys: ['Escape'],
             command: 'leave-insert-mode'
         });
+        commands.addKeyBinding({
+            selector: '.jp-Notebook:focus.jp-mod-commandMode',
+            keys: ['I'],
+            command: 'enter-insert-mode'
+        });
```

Comment:
I'm not if you need to have a custom command here. You can probably just use `notebook:enter-edit-mode`. Similar to jlab, I'd probably change the selector to `.jp-Notebook:focus`. Here's the shortcuts from jlab:

https://github.com/jupyterlab/jupyterlab/blob/master/packages/shortcuts-extension/schema/plugin.json#L599

Reply:
I think the extra command is necessary if we want <kbd>i</kbd> to go directly into Vim insert mode, which is the behavior I had in mind. Just using `notebook:enter-edit-mode` enters Vim command mode.

In any case, the delay persists even if I use `notebook:enter-edit-mode`, so maybe it has something to do with <kbd>i</kbd> as the key.

I did update the selector in my last commit.

Reply:
Sorry, somehow I read as this going into vim command mode, your code looks fine to me.

I think I'll try this out to see if I can identify anything.